aws/table_aws_ssm_document.go

```diff
@@ -119,8 +137,15 @@ func tableAwsSSMDocument(_ context.Context) *plugin.Table {
 			},
 			{
 				Name:        "owner",
+				Description: "The AWS user account type that created the document to filter results. Possible values: Self, Amazon, Public, Private, ThirdParty, All, Default.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("owner"),
+			},
+			{
+				Name:        "owner_output",
```

Comment:
@LalitTurbot What's an example of what `owner_output` returns? Is there a list of possible values? Is it the same set of valid values for the current `owner` column?